.travis.yml

```diff
@@ -11,29 +11,22 @@ julia:
   - 1.3
   - nightly
 
+before_script:
+  - git config --global user.name "GitUser"
```

Comment:
I think it is because windows builds complain if you use git on travis without these commands. I guess not directly useful here, I copy pasted the travis from a different project...

Reply:
ok maybe remove it then